docs/1 Introduction/3 Quickstart.md

```diff
@@ -22,7 +22,7 @@ starport serve
 
 The `serve` command installs dependencies, builds, initializes, and starts the blockchain.
 
-## Define
+## Add features
 
 To add a custom type with create, read, update, and delete (CRUD) functionality:
```

Comment:
module is top-level abstraction, encapsulates logic for a particular business domain 

- Interchain Exchange
- Chat
- Blog 

Scaffold types within modules. A module can have multiple types: orderbook, order, retire, and so on. the CRUD actions are specific to the type.